### PR TITLE
rough api added to Test.c

### PR DESCRIPTION
**Pull request by ndoubleg**

### CODE REVIEW

File: Test.C [hunk 1]
```diff
@@ -581,3 +590,86 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
...
+* Description: Example of curl calls to add player score
+* Parameters:  playerName, score, seed, version
+* Return:      void
+******************************************************************************/
+void addScore(char playerName, int score, int seed, int version){
+  CURL *curl;
+  CURLcode res;
+  
+  /* Creates the fields in which the scores are saved to the website */
+  sprintf(char *fields, "player_name=%s&score=%d&seed=%d&version=%d", &playerName, &score, &seed, &version);
```
Comment (Phabbits) [inline Test.C]: fields is a string or character array in C. `char *fields` is not how you initiate an array. Please initiate the array outside of the `sprintf` function. This is why you are getting `fields` was not declared in this scope on line 674

**Review by Phabbits — CHANGES_REQUESTED** [review]
Look over this.

File: Test.C (hunk 1 above)
Comment (ndoubleg) [inline Test.C]:
```suggestion
void addScore(char playerName[NAME_LENGTH], int score, int seed, int version){
  CURL *curl;
  CURLcode res;
  
  char fields[50];
  
  /* Creates the fields in which the scores are saved to the website */
  sprintf(fields, "player_name=%s&score=%d&seed=%d&version=%d", playerName, score, seed, version);
```

Comment (ndoubleg) [reply Test.C]: to print the fields, where exactly would the `printf("%s", fields);` go in this?
edit: was able to figure out.

File: Test.C [hunk 2]
```diff
@@ -581,3 +590,86 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
...
+* Description: Example of how to get scores from API
+* Parameters:  seed, version
+* Return:      void
+******************************************************************************/
+void getScores(int seed, int version){
+  CURL *curl;
+  CURLcode res;
+  
+  /* Creates the fields in which the scores saved on the website are called upon */
+  sprintf(char *fields, "seed=%d&version=%d", &seed, &version);
```
Comment (ndoubleg) [inline Test.C]:
```suggestion
  char fields[50];
  
  /* Creates the fields in which the scores saved on the website are called upon */
  sprintf(fields, "seed=%d&version=%d", seed, version);
```

File: Test.C [hunk 3]
```diff
@@ -388,6 +393,10 @@ int main(){
 
         // Save player score to leaderboard (prototype 2)
     }
+	
+	// Calling upon the function for adding/grabbing the score from the website
+	void addScore(char playerName, int score, int seed, int version);
+	void getScores(int seed, int version);
```
Comment (Phabbits) [inline Test.C]: We can just hardcode defaults right now.
```suggestion
	void getScores(1, 0.01);
```

File: Test.C [hunk 4]
```diff
@@ -388,6 +393,10 @@ int main(){
 
         // Save player score to leaderboard (prototype 2)
     }
+	
+	// Calling upon the function for adding/grabbing the score from the website
+	void addScore(char playerName, int score, int seed, int version);
```
Comment (Phabbits) [inline Test.C]: Lets just hardcode defaults for now.
```suggestion
	void addScore(player.name, player.currentCredits, 1, 0.01);
```

File: Test.C [hunk 5]
```diff
@@ -581,3 +590,86 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
...
+
+    /* always cleanup */
+    curl_easy_cleanup(curl);
+  }
+  curl_global_cleanup();
+  
+  /* Formatting for the display of the scores */
+  printf(" # | Player Name                  | Score  | Seed  | Version  \n");
+  printf("---|------------------------------|--------|-------|----------\n");
+  printf("player_name=%s&score=%d&seed=%d&version=%d", &playerName, &score, &seed, &version)
```
Comment (ndoubleg) [inline Test.C]:
```suggestion
```

**Review by Phabbits — CHANGES_REQUESTED** [review]

**Review by Phabbits — APPROVED** [review]

**Pull request merged**